etl/tests/sample.h2k

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="UTF-8" ?>
+<?xml version="1.0" encoding="UTF-8" ?>

```

Comment:
There seemed to be 3 invisible characters at the start of this line that `lxml` was able to ignore on unix based systems, but caused a crash on windows.

Reply:
O_o

Good job tracking that one down.